clang-tidy reports: unknown pragma ignored

```

/home/jenkins/agent/workspace/tiflash-build-common/tiflash/dbms/src/Common/TargetSpecific.h:559:1: error: unknown pragma ignored [clang-diagnostic-unknown-pragmas]

  559 | TIFLASH_TARGET_SPECIFIC_NAMESPACE(template <size_t LENGTH> struct Word {

      | ^

/home/jenkins/agent/workspace/tiflash-build-common/tiflash/dbms/src/Common/TargetSpecific.h:535:5: note: expanded from macro 'TIFLASH_TARGET_SPECIFIC_NAMESPACE'

  535 |     TIFLASH_AVX_NAMESPACE(__VA_ARGS__)         \

      |     ^

/home/jenkins/agent/workspace/tiflash-build-common/tiflash/dbms/src/Common/TargetSpecific.h:494:5: note: expanded from macro 'TIFLASH_AVX_NAMESPACE'

  494 |     TIFLASH_BEGIN_AVX_SPECIFIC_CODE           \

      |     ^

/home/jenkins/agent/workspace/tiflash-build-common/tiflash/dbms/src/Common/TargetSpecific.h:54:9: note: expanded from macro 'TIFLASH_BEGIN_AVX_SPECIFIC_CODE'

   54 |         _Pragma("clang attribute push(__attribute__((target(\"sse,sse2,sse3,ssse3,sse4,popcnt,avx,avx2\"))),apply_to=function)")

      |         ^

note: expanded from here
```

--- a/.clang-tidy
+++ b/.clang-tidy
@@ -208,3 +208,5 @@ CheckOptions:
     value: CamelCase
   - key: readability-identifier-naming.UsingCase
     value: CamelCase
+ExtraArgs:
+- -Wno-unknown-pragmas # remove with LLVM 18 https://github.com/llvm/llvm-project/issues/67380

